cloudpebble/settings.py

```diff
@@ -144,7 +145,7 @@
 )
 
 BOWER_INSTALLED_APPS = (
-    'https://github.com/krisk/Fuse.git#2c1560d763',
+    'https://github.com/krisk/Fuse.git',
```

Comment:
Thinking about it, I think I only actually tests this on Docker. But on the other hand it's working in the PR branch right now, so I assume it does?